src/providers/hover.ts

```diff
@@ -69,7 +69,13 @@ export async function provideHover(
                         if (text) {
                             const hover = new vscode.MarkdownString();
                             hover.isTrusted = true;
-                            hover.appendMarkdown(_.trim(text, '"'));
+                            hover.appendMarkdown(
+                                _.replace(
+                                    _.replace(_.trim(text, '"'), /\\n/g, '\n'),
+                                    /\\"/g,
+                                    '"'
+                                )
+                            );
```

Comment:
does anyone know of a better way to do this in js? I wanted to do it with a regex group, but "\$1" is not the same as "\n" even if $1 === "n", which is annoying.

Reply:
I don't think it's possible since the characters are `\` and `n`, $1 will never contain a newline.

Maybe something like this is clearer though:
```typescript
    const hoverText = text
        .replace(/^"|"$/g, '')
        .replace(/\\n/g, '\n')
        .replace(/\\"/g, '"');
    hover.appendMarkdown(hoverText);
```

(Untested)

Reply:
Yeah. I'll make a to-do for myself to see if I can somehow get the repl to provide me with non-escaped string output so we don't have to run this. But your version does look more readable. We need JS threading macros 🤯